api/HTMLEmbedElement.json

```diff
@@ -0,0 +1,58 @@
+{
+  "api": {
+    "HTMLEmbedElement": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLEmbedElement",
+        "support": {
+          "webview_android": {
+            "version_added": true,
+            "notes": "Starting with Chrome 58/Opera 45, this interface can no longer be called as a function."
+          },
+          "chrome": {
+            "version_added": true,
+            "notes": "Starting with Chrome 58, this interface can no longer be called as a function."
+          },
+          "chrome_android": {
+            "version_added": "Yes",
+            "notes": "Starting with Chrome 58, this interface can no longer be called as a function."
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "1",
+            "notes": "(Gecko 1.7 or earlier)"
```

Comment:
We stopped talking about Gecko on MDN and we disallow it in BCD. Please remove the note.